buildSrc/src/main/java/dependencies/Versions.kt

```diff
@@ -1,16 +1,16 @@
 package dependencies
 
 private object Versions {
-    val androidTargetSdkVersion = 28
-    val androidCompileSdkVersion = 28
-    val androidMinSdkVersion = 21
+    const val androidTargetSdkVersion = 29
+    const val androidCompileSdkVersion = 29
+    const val androidMinSdkVersion = 21
```

Comment:
I understand both opinions about cost.  👀
For example, Dep.kt has a problem that cost cannot be used because it refers to other variables.  Also, to keep many dependencies simple, we operate with no cost.  
Please let me know if you have any opinions🙏

Reply:
> For example, Dep.kt has a problem that cost cannot be used because it refers to other variables. 

→ Can all vals of Dep.kt be changed to const val ? ( except for hyperionPlugins )

but... I overlooked using arrays...
This change will revert 🙇